static/app/components/replays/videoReplayer.tsx

```diff
@@ -77,6 +77,9 @@ export class VideoReplayer {
     // Initially, only load some videos
     this.preloadVideos({low: 0, high: PRELOAD_BUFFER});
 
+    // Show first frame by default
+    this.showVideo(this._videos[0]);
```

Comment:
Would calling `loadSegment(0)` here be better instead? I think this could also take care of the above preload call as well. Then this would correctly set `this._currentIndex` in the loadeddata event listener as well.

```suggestion
    this.loadSegment(0);
```

Reply:
calling `loadSegment` doesn't work (no frame shows up) :( it also causes the bug we fixed here: https://github.com/getsentry/sentry/pull/67911

Reply:
Ahh